Microsoft.Research/ManagedContract.Setup/DotNet.Contracts.targets

```diff
@@ -0,0 +1,14 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <CodeContractsInstallDir>$(MSBuildThisFileDirectory)</CodeContractsInstallDir>
```

Comment:
I am of the same opinion as yaakov-h, I think that you add the nuget package in order to use that specific version and no other. 

We could relatively simple make it use the system installed version if one is installed and ignore the nuget version, (easier than the other way around), but I would find that un-intuitive and confusing.